doc/changes.md

```diff
@@ -1,5 +1,113 @@
 # Changelog
 
+
+## Release v0.8.0 (2022/12/28)
+
+[![Digital Object Identifier for PyGMT v0.8.0](https://zenodo.org/badge/DOI/10.5281/zenodo.7481934.svg)](https://doi.org/10.5281/zenodo.7481934)
+
+### Highlights
+
+* 🎉 **Eighth minor release of PyGMT** 🎉
+* Wrapped ternary
+* Easy access to various remote datasets
+* Add a intro tutorial for creating contour maps
+* Support tab auto-completion for all GMT defaults
```

Comment:
```suggestion
* Added support for tab auto-completion for all GMT default parameters
* Created functions to download GMT remote datasets
* Wrapped the ternary module
* Added an intro tutorial for creating contour maps
```

My take on how this should be ordered, since I think tab autocomplete is the biggest change, followed by the remote dataset functions. To keep it standardized, I think there should also be a verb of the same tense saying what happened (e.g. added, created, wrapped, etc.).